nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -413,6 +413,15 @@ public Description matchAssignment(AssignmentTree tree, VisitorState state) {
           state,
           ASTHelpers.getSymbol(tree.getVariable()));
     }
+    MethodTree initializerMethod = ASTHelpers.findEnclosingNode(state.getPath(), MethodTree.class);
+    if (initializerMethod != null) {
+      handler.serializeClassFieldInitializationInfo(
```

Comment:
I think this method should be called `handleFieldAssignment` or something like that, in case future handlers want to implement it for some other purpose

Reply:
37deec8c50ca71413509d3db4b3f5300357e9a9d